package.json

```diff
@@ -12,10 +12,10 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@dnd-kit/core": "^6.2.0",
-    "@dnd-kit/sortable": "^9.0.0",
+    "@hello-pangea/dnd": "^17.0.0",
     "@heroicons/react": "^2.2.0",
     "@serwist/next": "^9.0.10",
+    "@testing-library/user-event": "^14.5.2",
```

Comment:
I think this is a dev dependency, could you uninstall and reinstall as dev please? Sowrry!

Reply:
Done! Thank you for pointing that out